src/js/edu-benefits/actions/index.js

```diff
@@ -124,9 +129,20 @@ export function submitForm(data) {
     })
     .then(res => {
       if (res.ok) {
+        
+        window.dataLayer.push({
+          event: 'edu-submission-successful',
+          //submission-id: If you want to log anything here
```

Comment:
I don't think we need to log the submission ids here, since we're storing that in AWS.